.github/workflows/hyperfine.yml

```diff
@@ -156,8 +154,8 @@ jobs:
           done
         fi
         find target_programs -name '*.json' -exec basename -s .json '{}' '+' | \
-        sort | xargs -P 2 -I '{program}' \
-        hyperfine -N -r 10 --export-markdown "target_programs/{program}.md" \
+        sort | xargs -I '{program}' \
+        hyperfine -N -w 10 -r 20 --export-markdown "target_programs/{program}.md" \
```

Comment:
We should probably revert these to the original values so pipelines don't take too long.